keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -1056,6 +1149,7 @@ Major milestones might include:
 
 - 2023-02-10: Initial KEP published.
 - v1.31.0(2024-08-13): Alpha
+- v1.32.0: Beta (enabled by default)
```

Comment:
I apologize for not making this link before.

If this totally depends opn containerd 2.0 and 2.0 is NOT YET RELEASED, then I am wary of moving this to beta, especially being enabled-by-default.  That's a tripping hazard we don't need to introduce.

1) Are there any parts of this that containerd 1.x implements that could be advanced while leaving the policy in alpha?  e.g. the status stuff?

2) Is there an urgent need to move this to beta (for CRI-O, I assume)?  If so, could it be off-by-default (which is ~identical to alpha)?

Convince me I am over-reacting?

Reply:
No urgency from CRI-O side to move this to beta.

Reply:
I think the planning around version skew also needs more work, so it may be a good idea to postpone it.

Reply:
Don't get me wrong, I want this feature, but it seems like a bad idea to move features to beta where the majority (anecdotally) of kube users CAN'T POSSIBLY use it (because it's linked to containerd 2 and that is not yet released).

Are there any parts of this that containerd 1.x implements that could be advanced while leaving the policy in alpha? e.g. the status stuff?